src/main/java/lottery/Color.java

```diff
@@ -0,0 +1,10 @@
+package lottery;

```

Comment:
Why ball and color in lottery package is there are any logic to hold them together